NewForce-Cohort-5/tabloidfullstack-spring-of-deception
# cbg-ViewAllPosts

# Description
Created a post list.  A user can log in and see a link to posts at the top of the page.  Posts are ordered by most recent publish date.

## Type of change


# Testing Instructions

1. Clone down the repository.
2. Run `npm install` while in the root directory
3. `git fetch -a`
4. `git checkout cbg-ViewAllPosts`
5. Navigate to the root directory for this repo [tabloidfullstack-spring-of-deception](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception)
6. `start Tabloid.sln`
7. Make sure your db is created with seed data. Run [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/01_Tabloid_Create_DB.sql) and [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/02_Tabloid_Seed_Data.sql) queries in Visual Studio.
8. Start the server.
9. Navigate to [client](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/tree/main/Tabloid/client)
10. `npm start`
11. Log in as admin foo@bar.com (password doesn't matter, can be left blank).
12. Click Posts at the top of the page in the navbar
13. You should see a full list of posts ordered by my recent published date



# Checklist:
